frontend/src/employee-frontend/components/messages/MessageEditor.tsx

```diff
@@ -94,10 +102,19 @@ export default React.memo(function MessageEditor({
         <Gap size={'xs'} />
         <div>{i18n.messages.messageEditor.sender}</div>
         <Gap size={'xs'} />
-        <InputField
-          value={bulletin.sender}
-          onChange={(sender) => onChange({ sender })}
-          data-qa={'input-sender'}
+        <Select
+          options={mappedSenderOptions}
+          onChange={(selected) =>
+            selected &&
+            'value' in selected &&
+            onChange({ sender: selected.value })
+          }
+          value={
+            mappedSenderOptions.find(
+              ({ value }) => value === bulletin.sender
+            ) || mappedSenderOptions[0]
```

Comment:
Do we want to default to `mappedSenderOptions[0]` here? We could just let it default to `null` and force the user to choose their sender name each time and maybe reduce errors?